CONSOLE ERROR: line 77: Uncaught (in promise) TypeError: The stream has been aborted
This is a testharness.js-based test.
PASS Piping from an errored readable stream to an errored writable stream 
PASS Piping from an errored readable stream to an errored writable stream; preventAbort = true 
FAIL Piping from an errored readable stream to a closed writable stream assert_array_equals: lengths differ, expected 1 got 2
PASS Piping from a closed readable stream to an errored writable stream 
PASS Piping from a closed readable stream to a closed writable stream 
Harness: the test ran to completion.

